ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -1061,7 +1061,8 @@ def plot_subsets_hists(
                 output_filename_feature += ".png"
         plt.xlabel("ML qual", fontsize=font_size)
         plt.ylabel("Density", fontsize=font_size)
-        plt.yscale("log")
+        if logflag:
```

Comment:
where do you set logflag to True?

Reply:
you are right, this is not necessary anymore, I'll remove